eng/common/scripts/Verify-Links.ps1

```diff
@@ -286,9 +299,12 @@ while ($pageUrisToCheck.Count -ne 0)
   Write-Host "Found $($linkUris.Count) links on page $pageUri";
   $badLinksPerPage = @();
   foreach ($linkUri in $linkUris) {
-    $linkUri = ReplaceGithubLink $linkUri
-    $isLinkValid = CheckLink $linkUri
+    $replacedLink = ReplaceGithubLink $linkUri
+    $isLinkValid = CheckLink $replacedLink
     if (!$isLinkValid -and !$badLinksPerPage.Contains($linkUri)) {
+      if (!$linkUri.ToString().Trim()) {
+        $linkUri = $emptyLinkMessage
```

Comment:
Is this just to help with the error logging?

Reply:
We need to print something in summary. Here is a replacement for empty line. It looks weird if we print empty line here. It only gets print once per page.